src/main/java/com/google/idea/perf/Autocomplete.kt

```diff
@@ -117,7 +129,13 @@ class AutocompleteCompletionProvider: TextCompletionProvider {
             .map { it.source }
     }
 
+    private fun predictMethodToken(className: String, token: String): List<String> {
+        val clazz = Class.forName(className)
+        val methodNames = clazz.methods.map { it.name.substringAfter('$') }
```

Comment:
Just curious, why is `substringAfter('$')` needed?

Reply:
It removes `accessor$` prefixes. e.g. `com.google.idea.perf.TracerController` will list out methods such as `accessor$updateTracepointUi`.